services/gen-aws-conf.sh

```diff
@@ -4,7 +4,20 @@ set -e
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/" && pwd )"
 
-which yq >/dev/null || ( echo "*** please install yq ( https://github.com/mikefarah/yq ) in your path."; exit 22 )
+command -v yq >/dev/null || \
+    ( echo "*** please install yq ( https://github.com/mikefarah/yq ) in your path."; \
+      echo "    e.g.: wget https://github.com/mikefarah/yq/releases/download/2.1.2/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq"; \
+      echo "    NOTE: later versions may not work."; \
+      exit 22 \
+    )
+
+command -v awscli >/dev/null || \
```

Comment:
Shouldn't this be just `aws`? At least on my machine it's `aws`.

Reply:
you're right.  thanks!